modules/dreamview/frontend/src/store/websocket/websocket_realtime.js

```diff
@@ -1,14 +1,15 @@
 import STORE from "store";
 import RENDERER from "renderer";
 import MAP_NAVIGATOR from "components/Navigation/MapNavigator";
+import UTTERANCE from "store/utterance";
 import Worker from 'utils/webworker.js';
 
 export default class RosWebSocketEndpoint {
     constructor(serverAddr) {
         this.serverAddr = serverAddr;
         this.websocket = null;
         this.simWorldUpdatePeriodMs = 100;
-        this.simWorldLastUpdateTimestamp = 0;
+        this.simWorldLastUpdateTimestamp = new Date().getTime();
```

Comment:
checkMessage relies on the fact that this variable is initially 0. The connection lost alert can follow the same check.

Reply:
Good catch. Fixed the logic.